client/store/actions/events.ts

```diff
@@ -0,0 +1,49 @@
+import { ActionCreator } from 'redux';
+import { HttpService } from 'client/services/http-service';
+import { eventsTypes } from '../types';
+
+/****************
+ * Actions
+ ****************/
+export const fetchStart = (): eventsTypes.IEventActionTypes => {
+  return {
+    type: eventsTypes.FETCH_START,
+  };
+};
+
+export const fetchSuccess = (
+  events: eventsTypes.IEventModal[],
+  chapterId: string,
+): eventsTypes.IEventActionTypes => {
+  return {
+    type: eventsTypes.FETCH_SUCCESS,
+    payload: { events, chapterId },
+  };
+};
+
+export const fetchFail = (error: string): eventsTypes.IEventActionTypes => {
+  return {
+    payload: error,
```

Comment:
not an issue: just we should have consistency 😅in return statement

Reply:
So just change the order?

Reply:
yes, I mean order should be consistent. code looks good 😬